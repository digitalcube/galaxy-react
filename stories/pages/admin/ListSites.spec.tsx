import { render } from '@testing-library/react';
import React from 'react';
import { Default as ListSites } from './ListSites.stories';

describe('ListSites', () => {
  it('to match snapshot(default)', () => {
    const { asFragment } = render(<ListSites {...(ListSites.args as any)} />);
    expect(asFragment());
  });
});
