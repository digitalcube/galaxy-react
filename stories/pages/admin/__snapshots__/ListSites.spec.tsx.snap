// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ListSites to match snapshot(default) 1`] = `
<DocumentFragment>
  <div
    class="d-flex flex-column min-vh-100 shifter-dashboard"
  >
    <nav
      class="navbar navbar-expand-lg"
    >
      <a
        class="navbar-brand"
        href="/"
      >
        <img
          alt="Shifter"
          height="32"
          width="24"
        />
      </a>
      <form
        class="form-inline my-2 my-lg-0 flex-grow-1 d-flex position-relative form-search"
      >
        <div
          class="input-group"
        >
          <svg
            class="fas fa-search form-search-icon position-absolute"
            fill="currentColor"
            height="1em"
            stroke="currentColor"
            stroke-width="0"
            viewBox="0 0 512 512"
            width="1em"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M505 442.7L405.3 343c-4.5-4.5-10.6-7-17-7H372c27.6-35.3 44-79.7 44-128C416 93.1 322.9 0 208 0S0 93.1 0 208s93.1 208 208 208c48.3 0 92.7-16.4 128-44v16.3c0 6.4 2.5 12.5 7 17l99.7 99.7c9.4 9.4 24.6 9.4 33.9 0l28.3-28.3c9.4-9.4 9.4-24.6.1-34zM208 336c-70.7 0-128-57.2-128-128 0-70.7 57.2-128 128-128 70.7 0 128 57.2 128 128 0 70.7-57.2 128-128 128z"
            />
          </svg>
          <input
            aria-label="Search"
            class="form-control"
            placeholder="search"
            type="text"
            value=""
          />
        </div>
      </form>
      <button
        aria-label="Toggle navigation"
        class="navbar-toggler"
        type="button"
      >
        <span
          class="navbar-toggler-icon"
        />
      </button>
      <div
        class="navbar-collapse flex-grow-0 collapse"
      >
        <ul
          class="ml-auto navbar-nav nav"
        >
          <li
            class="active nav-item"
          >
            <a
              class="rounded font-weight-bold nav-link"
              to="#"
            >
              Sites
            </a>
          </li>
          <li
            class="nav-item"
          >
            <a
              class="rounded font-weight-bold nav-link"
              to="#"
            >
              Teams
            </a>
          </li>
          <li
            class="nav-item"
          >
            <a
              class="rounded font-weight-bold nav-link"
              to="#"
            >
              Guides
            </a>
          </li>
          <li
            class="dropdown nav-item"
          >
            <a
              aria-expanded="false"
              aria-haspopup="true"
              class="nav-link font-weight-bold dropdown-toggle nav-link"
              data-toggle="dropdown"
              href="#"
              role="button"
            >
              <img
                alt="Gravatar for test@example.com"
                class="react-gravatar p-0  rounded-circle"
                height="32"
                src="//www.gravatar.com/avatar/55502f40dc8b7c769880b10874abc9d0?d=retro&r=g&s=32"
                srcset="//www.gravatar.com/avatar/55502f40dc8b7c769880b10874abc9d0?d=retro&r=g&s=64 2x"
                width="32"
              />
            </a>
            <div
              aria-hidden="true"
              class="dropdown-menu dropdown-menu-right"
              role="menu"
              tabindex="-1"
            >
              <button
                class="dropdown-item"
                role="menuitem"
                tabindex="0"
                type="button"
              >
                <a
                  class="dropdown-item px-3 py-2 d-flex align-items-center justify-content-between"
                  href="#"
                  to="#"
                >
                  Account
                  <svg
                    fill="currentColor"
                    height="1em"
                    stroke="currentColor"
                    stroke-width="0"
                    viewBox="0 0 448 512"
                    width="1em"
                    xmlns="http://www.w3.org/2000/svg"
                  >
                    <path
                      d="M224 256c70.7 0 128-57.3 128-128S294.7 0 224 0 96 57.3 96 128s57.3 128 128 128zm89.6 32h-16.7c-22.2 10.2-46.9 16-72.9 16s-50.6-5.8-72.9-16h-16.7C60.2 288 0 348.2 0 422.4V464c0 26.5 21.5 48 48 48h352c26.5 0 48-21.5 48-48v-41.6c0-74.2-60.2-134.4-134.4-134.4z"
                    />
                  </svg>
                </a>
              </button>
              <button
                class="dropdown-item"
                role="menuitem"
                tabindex="0"
                type="button"
              >
                <a
                  class="dropdown-item px-3 py-2 d-flex align-items-center justify-content-between"
                  href="#"
                  to="#"
                >
                  Log out 
                  <svg
                    fill="currentColor"
                    height="1em"
                    stroke="currentColor"
                    stroke-width="0"
                    viewBox="0 0 512 512"
                    width="1em"
                    xmlns="http://www.w3.org/2000/svg"
                  >
                    <path
                      d="M497 273L329 441c-15 15-41 4.5-41-17v-96H152c-13.3 0-24-10.7-24-24v-96c0-13.3 10.7-24 24-24h136V88c0-21.4 25.9-32 41-17l168 168c9.3 9.4 9.3 24.6 0 34zM192 436v-40c0-6.6-5.4-12-12-12H96c-17.7 0-32-14.3-32-32V160c0-17.7 14.3-32 32-32h84c6.6 0 12-5.4 12-12V76c0-6.6-5.4-12-12-12H96c-53 0-96 43-96 96v192c0 53 43 96 96 96h84c6.6 0 12-5.4 12-12z"
                    />
                  </svg>
                </a>
              </button>
            </div>
          </li>
        </ul>
      </div>
    </nav>
    <main
      class="px-4 sites-list"
    >
      <header
        class="d-flex align-items-center justify-content-start sites-list-header"
      >
        <h1
          class="mb-0 font-weight-bold sites-list-title"
        >
          Sites
        </h1>
        <div
          class="ml-3 list-sorter"
        >
          <select
            class="selectpicker show-tick"
            id=""
            name=""
          >
            <option
              data-content="<span class='option-item font-weight-bold'>Name:</span> A-Z"
            >
              Name: A-Z
            </option>
            <option
              data-content="<span class='option-item font-weight-bold'>Name:</span> Z-A"
            >
              Name: Z-A
            </option>
            <option
              data-content="<span class='option-item font-weight-bold'>Type:</span> Static"
            >
              Type: Static
            </option>
            <option
              data-content="<span class='option-item font-weight-bold'>Type:</span> Headless"
            >
              Type: Headless
            </option>
          </select>
        </div>
        <div
          class="ml-auto list-add"
        >
          <button
            class="px-3 py-0 d-flex align-items-center btn-colored font-weight-bold rounded btn-smaller btn btn-secondary"
            type="button"
          >
            <i
              aria-hidden="true"
              class="mr-2"
            >
              <svg
                class="d-block"
                fill="currentColor"
                height="1em"
                stroke="currentColor"
                stroke-width="0"
                viewBox="0 0 448 512"
                width="1em"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M416 208H272V64c0-17.67-14.33-32-32-32h-32c-17.67 0-32 14.33-32 32v144H32c-17.67 0-32 14.33-32 32v32c0 17.67 14.33 32 32 32h144v144c0 17.67 14.33 32 32 32h32c17.67 0 32-14.33 32-32V304h144c17.67 0 32-14.33 32-32v-32c0-17.67-14.33-32-32-32z"
                />
              </svg>
            </i>
            Create a new site
          </button>
        </div>
      </header>
      <section
        class="d-flex flex-column site-list-body"
      >
        <div
          class="mb-3 rounded d-md-flex align-items-start site-list-item"
        >
          <div
            class="progress undefined"
          >
            <div
              aria-valuemax="100"
              aria-valuemin="1"
              aria-valuenow="10"
              class="progress-bar"
              role="progressbar"
              style="width: 10.1010101010101%;"
            />
          </div>
          <div
            class="site-thumbnail rounded-left"
          >
            <img
              alt="Site"
              height="111"
              src="/images/site-a.png"
              width="167"
            />
          </div>
          <div
            class="pl-3 py-3 site-data"
          >
            <h2
              class="mb-0 font-weight-bold site-name"
            >
              Stable Value Investment Ltd.
            </h2>
            <div
              class="mt-3 d-flex align-items-center site-info"
            >
              <span
                class="d-inline-block align-middle site-url"
              >
                <a
                  href="stablevalueinvestment.org"
                >
                  stablevalueinvestment.org
                </a>
              </span>
              <span
                class="d-inline-block align-middle text-truncate site-description"
              >
                <span
                  class="mr-2 d-inline-block rounded-circle font-weight-bold text-center site-initial"
                >
                  D
                </span>
                <p
                  class="mb-0 d-inline site-description"
                >
                  super log name toshow a line break super long names work
                </p>
              </span>
            </div>
          </div>
          <div
            class="ml-auto pt-4 pr-3 d-flex align-items-center flex-shrink-0 site-setting"
          >
            <div
              class="ml-4 site-status"
            >
              <div
                class="rounded font-weight-bold c-site-status c-is-site-running"
              >
                WordPress Running
              </div>
            </div>
            <div
              class="ml-4 site-panel"
            >
              <a
                href="#"
                title="管理画面"
              >
                <i
                  aria-hidden="true"
                >
                  <svg
                    class="d-block"
                    fill="currentColor"
                    height="1em"
                    stroke="currentColor"
                    stroke-width="0"
                    viewBox="0 0 512 512"
                    width="1em"
                    xmlns="http://www.w3.org/2000/svg"
                  >
                    <path
                      d="M256 8C119.3 8 8 119.2 8 256c0 136.7 111.3 248 248 248s248-111.3 248-248C504 119.2 392.7 8 256 8zM33 256c0-32.3 6.9-63 19.3-90.7l106.4 291.4C84.3 420.5 33 344.2 33 256zm223 223c-21.9 0-43-3.2-63-9.1l66.9-194.4 68.5 187.8c.5 1.1 1 2.1 1.6 3.1-23.1 8.1-48 12.6-74 12.6zm30.7-327.5c13.4-.7 25.5-2.1 25.5-2.1 12-1.4 10.6-19.1-1.4-18.4 0 0-36.1 2.8-59.4 2.8-21.9 0-58.7-2.8-58.7-2.8-12-.7-13.4 17.7-1.4 18.4 0 0 11.4 1.4 23.4 2.1l34.7 95.2L200.6 393l-81.2-241.5c13.4-.7 25.5-2.1 25.5-2.1 12-1.4 10.6-19.1-1.4-18.4 0 0-36.1 2.8-59.4 2.8-4.2 0-9.1-.1-14.4-.3C109.6 73 178.1 33 256 33c58 0 110.9 22.2 150.6 58.5-1-.1-1.9-.2-2.9-.2-21.9 0-37.4 19.1-37.4 39.6 0 18.4 10.6 33.9 21.9 52.3 8.5 14.8 18.4 33.9 18.4 61.5 0 19.1-7.3 41.2-17 72.1l-22.2 74.3-80.7-239.6zm81.4 297.2l68.1-196.9c12.7-31.8 17-57.2 17-79.9 0-8.2-.5-15.8-1.5-22.9 17.4 31.8 27.3 68.2 27.3 107 0 82.3-44.6 154.1-110.9 192.7z"
                    />
                  </svg>
                </i>
              </a>
            </div>
            <div
              class="ml-4 font-weight-bold site-manage"
            >
              <a
                href="#"
              >
                Manage site
              </a>
            </div>
            <div
              class="ml-4 site-control"
            >
              <a
                aria-expanded="false"
                aria-haspopup="true"
                class="dropdown-toggle"
                data-toggle="dropdown"
                href="#"
                id="navbarDropdown"
                role="button"
              >
                <i
                  aria-hidden="true"
                >
                  <svg
                    class="d-block"
                    fill="currentColor"
                    height="1em"
                    stroke="currentColor"
                    stroke-width="0"
                    viewBox="0 0 512 512"
                    width="1em"
                    xmlns="http://www.w3.org/2000/svg"
                  >
                    <path
                      d="M328 256c0 39.8-32.2 72-72 72s-72-32.2-72-72 32.2-72 72-72 72 32.2 72 72zm104-72c-39.8 0-72 32.2-72 72s32.2 72 72 72 72-32.2 72-72-32.2-72-72-72zm-352 0c-39.8 0-72 32.2-72 72s32.2 72 72 72 72-32.2 72-72-32.2-72-72-72z"
                    />
                  </svg>
                </i>
              </a>
              <div
                aria-labelledby="navbarDropdown"
                class="dropdown-menu dropdown-menu-right"
              >
                <a
                  class="dropdown-item px-3 py-2 d-flex align-items-center justify-content-between"
                  href="#"
                >
                  Start WordPress
                  <i
                    aria-hidden="true"
                  >
                    <svg
                      class="d-block"
                      fill="currentColor"
                      height="1em"
                      stroke="currentColor"
                      stroke-width="0"
                      viewBox="0 0 448 512"
                      width="1em"
                      xmlns="http://www.w3.org/2000/svg"
                    >
                      <path
                        d="M424.4 214.7L72.4 6.6C43.8-10.3 0 6.1 0 47.9V464c0 37.5 40.7 60.1 72.4 41.3l352-208c31.4-18.5 31.5-64.1 0-82.6z"
                      />
                    </svg>
                  </i>
                </a>
                <a
                  class="dropdown-item px-3 py-2 d-flex align-items-center justify-content-between"
                  href="#"
                >
                  Restart WordPress
                  <i
                    aria-hidden="true"
                  >
                    <svg
                      class="d-block"
                      fill="currentColor"
                      height="1em"
                      stroke="currentColor"
                      stroke-width="0"
                      viewBox="0 0 448 512"
                      width="1em"
                      xmlns="http://www.w3.org/2000/svg"
                    >
                      <path
                        d="M400 32H48C21.5 32 0 53.5 0 80v352c0 26.5 21.5 48 48 48h352c26.5 0 48-21.5 48-48V80c0-26.5-21.5-48-48-48z"
                      />
                    </svg>
                  </i>
                </a>
              </div>
            </div>
          </div>
        </div>
        <div
          class="mb-3 rounded d-md-flex align-items-start site-list-item"
        >
          <div
            class="progress"
          >
            <div
              aria-valuemax="100"
              aria-valuemin="0"
              aria-valuenow="0"
              class="progress-bar"
              role="progressbar"
            />
          </div>
          <div
            class="site-thumbnail rounded-left"
          >
            <img
              alt=""
              height="111"
              src="/images/site-b.png"
              width="167"
            />
          </div>
          <div
            class="pl-3 py-3 site-data"
          >
            <h2
              class="mb-0 font-weight-bold site-name"
            >
              Example Site
            </h2>
            <div
              class="d-flex align-items-center site-info"
            >
              <span
                class="d-inline-block align-middle site-url"
              >
                <a
                  href="nervous-lamarr9210.on.getshifter.io"
                >
                  nervous-lamarr9210.on.getshifter.io
                </a>
              </span>
              <span
                class="d-inline-block align-middle text-truncate"
              >
                <span
                  class="mr-2 d-inline-block rounded-circle font-weight-bold text-center site-initial"
                >
                  D
                </span>
                <p
                  class="mb-0 d-inline site-description"
                >
                  Team name
                </p>
              </span>
            </div>
          </div>
          <div
            class="ml-auto pt-4 pr-3 d-flex align-items-center flex-shrink-0 site-setting"
          >
            <div
              class="ml-4 site-status"
            >
              <div
                class="rounded font-weight-bold c-site-status c-is-site-stopped"
              >
                WordPress Stopped
              </div>
            </div>
            <div
              class="ml-4 site-panel"
            >
              <a
                href="#"
                title="管理画面"
              >
                <i
                  aria-hidden="true"
                >
                  <svg
                    class="d-block"
                    fill="currentColor"
                    height="1em"
                    stroke="currentColor"
                    stroke-width="0"
                    viewBox="0 0 512 512"
                    width="1em"
                    xmlns="http://www.w3.org/2000/svg"
                  >
                    <path
                      d="M256 8C119.3 8 8 119.2 8 256c0 136.7 111.3 248 248 248s248-111.3 248-248C504 119.2 392.7 8 256 8zM33 256c0-32.3 6.9-63 19.3-90.7l106.4 291.4C84.3 420.5 33 344.2 33 256zm223 223c-21.9 0-43-3.2-63-9.1l66.9-194.4 68.5 187.8c.5 1.1 1 2.1 1.6 3.1-23.1 8.1-48 12.6-74 12.6zm30.7-327.5c13.4-.7 25.5-2.1 25.5-2.1 12-1.4 10.6-19.1-1.4-18.4 0 0-36.1 2.8-59.4 2.8-21.9 0-58.7-2.8-58.7-2.8-12-.7-13.4 17.7-1.4 18.4 0 0 11.4 1.4 23.4 2.1l34.7 95.2L200.6 393l-81.2-241.5c13.4-.7 25.5-2.1 25.5-2.1 12-1.4 10.6-19.1-1.4-18.4 0 0-36.1 2.8-59.4 2.8-4.2 0-9.1-.1-14.4-.3C109.6 73 178.1 33 256 33c58 0 110.9 22.2 150.6 58.5-1-.1-1.9-.2-2.9-.2-21.9 0-37.4 19.1-37.4 39.6 0 18.4 10.6 33.9 21.9 52.3 8.5 14.8 18.4 33.9 18.4 61.5 0 19.1-7.3 41.2-17 72.1l-22.2 74.3-80.7-239.6zm81.4 297.2l68.1-196.9c12.7-31.8 17-57.2 17-79.9 0-8.2-.5-15.8-1.5-22.9 17.4 31.8 27.3 68.2 27.3 107 0 82.3-44.6 154.1-110.9 192.7z"
                    />
                  </svg>
                </i>
              </a>
            </div>
            <div
              class="ml-4 font-weight-bold site-manage"
            >
              <a
                href="#"
              >
                Manage site
              </a>
            </div>
            <div
              class="ml-4 site-control"
            >
              <a
                aria-expanded="false"
                aria-haspopup="true"
                class="dropdown-toggle"
                data-toggle="dropdown"
                href="#"
                id="navbarDropdown"
                role="button"
              >
                <i
                  aria-hidden="true"
                >
                  <svg
                    class="d-block"
                    fill="currentColor"
                    height="1em"
                    stroke="currentColor"
                    stroke-width="0"
                    viewBox="0 0 512 512"
                    width="1em"
                    xmlns="http://www.w3.org/2000/svg"
                  >
                    <path
                      d="M328 256c0 39.8-32.2 72-72 72s-72-32.2-72-72 32.2-72 72-72 72 32.2 72 72zm104-72c-39.8 0-72 32.2-72 72s32.2 72 72 72 72-32.2 72-72-32.2-72-72-72zm-352 0c-39.8 0-72 32.2-72 72s32.2 72 72 72 72-32.2 72-72-32.2-72-72-72z"
                    />
                  </svg>
                </i>
              </a>
              <div
                aria-labelledby="navbarDropdown"
                class="dropdown-menu dropdown-menu-right"
              >
                <a
                  class="dropdown-item px-3 py-2 d-flex align-items-center justify-content-between"
                  href="#"
                >
                  Start WordPress
                  <i
                    aria-hidden="true"
                  >
                    <svg
                      class="d-block"
                      fill="currentColor"
                      height="1em"
                      stroke="currentColor"
                      stroke-width="0"
                      viewBox="0 0 448 512"
                      width="1em"
                      xmlns="http://www.w3.org/2000/svg"
                    >
                      <path
                        d="M424.4 214.7L72.4 6.6C43.8-10.3 0 6.1 0 47.9V464c0 37.5 40.7 60.1 72.4 41.3l352-208c31.4-18.5 31.5-64.1 0-82.6z"
                      />
                    </svg>
                  </i>
                </a>
                <a
                  class="dropdown-item px-3 py-2 d-flex align-items-center justify-content-between"
                  href="#"
                >
                  Restart WordPress
                  <i
                    aria-hidden="true"
                  >
                    <svg
                      class="d-block"
                      fill="currentColor"
                      height="1em"
                      stroke="currentColor"
                      stroke-width="0"
                      viewBox="0 0 448 512"
                      width="1em"
                      xmlns="http://www.w3.org/2000/svg"
                    >
                      <path
                        d="M400 32H48C21.5 32 0 53.5 0 80v352c0 26.5 21.5 48 48 48h352c26.5 0 48-21.5 48-48V80c0-26.5-21.5-48-48-48z"
                      />
                    </svg>
                  </i>
                </a>
              </div>
            </div>
          </div>
        </div>
        <div
          class="mb-3 rounded d-md-flex align-items-start site-list-item"
        >
          <div
            class="progress"
          >
            <div
              aria-valuemax="100"
              aria-valuemin="0"
              aria-valuenow="0"
              class="progress-bar"
              role="progressbar"
            />
          </div>
          <div
            class="site-thumbnail rounded-left"
          >
            <img
              alt=""
              height="111"
              src="/images/site-c.png"
              width="167"
            />
          </div>
          <div
            class="pl-3 py-3 site-data"
          >
            <h2
              class="mb-0 font-weight-bold site-name"
            >
              Interior Design Blog
            </h2>
            <div
              class="mt-3 d-flex align-items-center site-info"
            >
              <span
                class="d-inline-block align-middle site-url"
              >
                <a
                  href="interiordesignwebsite.com/blog/"
                >
                  interiordesignwebsite.com/blog/
                </a>
              </span>
              <span
                class="d-inline-block align-middle text-truncate site-description"
              >
                <span
                  class="mr-2 d-inline-block rounded-circle font-weight-bold text-center site-initial"
                >
                  D
                </span>
                <p
                  class="mb-0 d-inline site-description"
                >
                  Super long name to show a line break super long names work
                </p>
              </span>
            </div>
          </div>
          <div
            class="ml-auto pt-4 pr-3 d-flex align-items-center flex-shrink-0 site-setting"
          >
            <div
              class="ml-4 site-status"
            >
              <div
                class="rounded font-weight-bold c-site-status c-is-site-stopped"
              >
                WordPress Stopped
              </div>
            </div>
            <div
              class="ml-4 site-panel"
            >
              <a
                href="#"
                title="管理画面"
              >
                <i
                  aria-hidden="true"
                >
                  <svg
                    class="d-block"
                    fill="currentColor"
                    height="1em"
                    stroke="currentColor"
                    stroke-width="0"
                    viewBox="0 0 512 512"
                    width="1em"
                    xmlns="http://www.w3.org/2000/svg"
                  >
                    <path
                      d="M256 8C119.3 8 8 119.2 8 256c0 136.7 111.3 248 248 248s248-111.3 248-248C504 119.2 392.7 8 256 8zM33 256c0-32.3 6.9-63 19.3-90.7l106.4 291.4C84.3 420.5 33 344.2 33 256zm223 223c-21.9 0-43-3.2-63-9.1l66.9-194.4 68.5 187.8c.5 1.1 1 2.1 1.6 3.1-23.1 8.1-48 12.6-74 12.6zm30.7-327.5c13.4-.7 25.5-2.1 25.5-2.1 12-1.4 10.6-19.1-1.4-18.4 0 0-36.1 2.8-59.4 2.8-21.9 0-58.7-2.8-58.7-2.8-12-.7-13.4 17.7-1.4 18.4 0 0 11.4 1.4 23.4 2.1l34.7 95.2L200.6 393l-81.2-241.5c13.4-.7 25.5-2.1 25.5-2.1 12-1.4 10.6-19.1-1.4-18.4 0 0-36.1 2.8-59.4 2.8-4.2 0-9.1-.1-14.4-.3C109.6 73 178.1 33 256 33c58 0 110.9 22.2 150.6 58.5-1-.1-1.9-.2-2.9-.2-21.9 0-37.4 19.1-37.4 39.6 0 18.4 10.6 33.9 21.9 52.3 8.5 14.8 18.4 33.9 18.4 61.5 0 19.1-7.3 41.2-17 72.1l-22.2 74.3-80.7-239.6zm81.4 297.2l68.1-196.9c12.7-31.8 17-57.2 17-79.9 0-8.2-.5-15.8-1.5-22.9 17.4 31.8 27.3 68.2 27.3 107 0 82.3-44.6 154.1-110.9 192.7z"
                    />
                  </svg>
                </i>
              </a>
            </div>
            <div
              class="ml-4 font-weight-bold site-manage"
            >
              <a
                href="#"
              >
                Manage site
              </a>
            </div>
            <div
              class="ml-4 site-control"
            >
              <a
                aria-expanded="false"
                aria-haspopup="true"
                class="dropdown-toggle"
                data-toggle="dropdown"
                href="#"
                id="navbarDropdown"
                role="button"
              >
                <i
                  aria-hidden="true"
                >
                  <svg
                    class="d-block"
                    fill="currentColor"
                    height="1em"
                    stroke="currentColor"
                    stroke-width="0"
                    viewBox="0 0 512 512"
                    width="1em"
                    xmlns="http://www.w3.org/2000/svg"
                  >
                    <path
                      d="M328 256c0 39.8-32.2 72-72 72s-72-32.2-72-72 32.2-72 72-72 72 32.2 72 72zm104-72c-39.8 0-72 32.2-72 72s32.2 72 72 72 72-32.2 72-72-32.2-72-72-72zm-352 0c-39.8 0-72 32.2-72 72s32.2 72 72 72 72-32.2 72-72-32.2-72-72-72z"
                    />
                  </svg>
                </i>
              </a>
              <div
                aria-labelledby="navbarDropdown"
                class="dropdown-menu dropdown-menu-right"
              >
                <a
                  class="dropdown-item px-3 py-2 d-flex align-items-center justify-content-between"
                  href="#"
                >
                  Start WordPress
                  <i
                    aria-hidden="true"
                  >
                    <svg
                      class="d-block"
                      fill="currentColor"
                      height="1em"
                      stroke="currentColor"
                      stroke-width="0"
                      viewBox="0 0 448 512"
                      width="1em"
                      xmlns="http://www.w3.org/2000/svg"
                    >
                      <path
                        d="M424.4 214.7L72.4 6.6C43.8-10.3 0 6.1 0 47.9V464c0 37.5 40.7 60.1 72.4 41.3l352-208c31.4-18.5 31.5-64.1 0-82.6z"
                      />
                    </svg>
                  </i>
                </a>
                <a
                  class="dropdown-item px-3 py-2 d-flex align-items-center justify-content-between"
                  href="#"
                >
                  Restart WordPress
                  <i
                    aria-hidden="true"
                  >
                    <svg
                      class="d-block"
                      fill="currentColor"
                      height="1em"
                      stroke="currentColor"
                      stroke-width="0"
                      viewBox="0 0 448 512"
                      width="1em"
                      xmlns="http://www.w3.org/2000/svg"
                    >
                      <path
                        d="M400 32H48C21.5 32 0 53.5 0 80v352c0 26.5 21.5 48 48 48h352c26.5 0 48-21.5 48-48V80c0-26.5-21.5-48-48-48z"
                      />
                    </svg>
                  </i>
                </a>
              </div>
            </div>
          </div>
        </div>
        <div
          class="mb-3 rounded d-md-flex align-items-start site-list-item"
        >
          <div
            class="progress"
          >
            <div
              aria-valuemax="100"
              aria-valuemin="0"
              aria-valuenow="0"
              class="progress-bar"
              role="progressbar"
            />
          </div>
          <div
            class="site-thumbnail rounded-left"
          >
            <img
              alt=""
              height="111"
              src="/images/site-d.png"
              width="167"
            />
          </div>
          <div
            class="pl-3 py-3 site-data"
          >
            <h2
              class="mb-0 font-weight-bold site-name"
            >
              Interior Design Blog
            </h2>
            <div
              class="mt-3 d-flex align-items-center site-info"
            >
              <span
                class="d-inline-block align-middle site-url"
              >
                <a
                  href="interiordesignwebsite.com/blog/"
                >
                  interiordesignwebsite.com/blog/
                </a>
              </span>
              <span
                class="d-inline-block align-middle text-truncate site-description"
              >
                <span
                  class="mr-2 d-inline-block rounded-circle font-weight-bold text-center site-initial"
                >
                  D
                </span>
                <p
                  class="mb-0 d-inline site-description"
                >
                  Super long name to show a line break super long names work
                </p>
              </span>
            </div>
          </div>
          <div
            class="ml-auto pt-4 pr-3 d-flex align-items-center flex-shrink-0 site-setting"
          >
            <div
              class="ml-4 site-status"
            >
              <div
                class="rounded font-weight-bold c-site-status c-is-site-stopped"
              >
                WordPress Stopped
              </div>
            </div>
            <div
              class="ml-4 site-panel"
            >
              <a
                href="#"
                title="管理画面"
              >
                <i
                  aria-hidden="true"
                >
                  <svg
                    class="d-block"
                    fill="currentColor"
                    height="1em"
                    stroke="currentColor"
                    stroke-width="0"
                    viewBox="0 0 512 512"
                    width="1em"
                    xmlns="http://www.w3.org/2000/svg"
                  >
                    <path
                      d="M256 8C119.3 8 8 119.2 8 256c0 136.7 111.3 248 248 248s248-111.3 248-248C504 119.2 392.7 8 256 8zM33 256c0-32.3 6.9-63 19.3-90.7l106.4 291.4C84.3 420.5 33 344.2 33 256zm223 223c-21.9 0-43-3.2-63-9.1l66.9-194.4 68.5 187.8c.5 1.1 1 2.1 1.6 3.1-23.1 8.1-48 12.6-74 12.6zm30.7-327.5c13.4-.7 25.5-2.1 25.5-2.1 12-1.4 10.6-19.1-1.4-18.4 0 0-36.1 2.8-59.4 2.8-21.9 0-58.7-2.8-58.7-2.8-12-.7-13.4 17.7-1.4 18.4 0 0 11.4 1.4 23.4 2.1l34.7 95.2L200.6 393l-81.2-241.5c13.4-.7 25.5-2.1 25.5-2.1 12-1.4 10.6-19.1-1.4-18.4 0 0-36.1 2.8-59.4 2.8-4.2 0-9.1-.1-14.4-.3C109.6 73 178.1 33 256 33c58 0 110.9 22.2 150.6 58.5-1-.1-1.9-.2-2.9-.2-21.9 0-37.4 19.1-37.4 39.6 0 18.4 10.6 33.9 21.9 52.3 8.5 14.8 18.4 33.9 18.4 61.5 0 19.1-7.3 41.2-17 72.1l-22.2 74.3-80.7-239.6zm81.4 297.2l68.1-196.9c12.7-31.8 17-57.2 17-79.9 0-8.2-.5-15.8-1.5-22.9 17.4 31.8 27.3 68.2 27.3 107 0 82.3-44.6 154.1-110.9 192.7z"
                    />
                  </svg>
                </i>
              </a>
            </div>
            <div
              class="ml-4 font-weight-bold site-manage"
            >
              <a
                href="#"
              >
                Manage site
              </a>
            </div>
            <div
              class="ml-4 site-control"
            >
              <a
                aria-expanded="false"
                aria-haspopup="true"
                class="dropdown-toggle"
                data-toggle="dropdown"
                href="#"
                id="navbarDropdown"
                role="button"
              >
                <i
                  aria-hidden="true"
                >
                  <svg
                    class="d-block"
                    fill="currentColor"
                    height="1em"
                    stroke="currentColor"
                    stroke-width="0"
                    viewBox="0 0 512 512"
                    width="1em"
                    xmlns="http://www.w3.org/2000/svg"
                  >
                    <path
                      d="M328 256c0 39.8-32.2 72-72 72s-72-32.2-72-72 32.2-72 72-72 72 32.2 72 72zm104-72c-39.8 0-72 32.2-72 72s32.2 72 72 72 72-32.2 72-72-32.2-72-72-72zm-352 0c-39.8 0-72 32.2-72 72s32.2 72 72 72 72-32.2 72-72-32.2-72-72-72z"
                    />
                  </svg>
                </i>
              </a>
              <div
                aria-labelledby="navbarDropdown"
                class="dropdown-menu dropdown-menu-right"
              >
                <a
                  class="dropdown-item px-3 py-2 d-flex align-items-center justify-content-between"
                  href="#"
                >
                  Start WordPress
                  <i
                    aria-hidden="true"
                  >
                    <svg
                      class="d-block"
                      fill="currentColor"
                      height="1em"
                      stroke="currentColor"
                      stroke-width="0"
                      viewBox="0 0 448 512"
                      width="1em"
                      xmlns="http://www.w3.org/2000/svg"
                    >
                      <path
                        d="M424.4 214.7L72.4 6.6C43.8-10.3 0 6.1 0 47.9V464c0 37.5 40.7 60.1 72.4 41.3l352-208c31.4-18.5 31.5-64.1 0-82.6z"
                      />
                    </svg>
                  </i>
                </a>
                <a
                  class="dropdown-item px-3 py-2 d-flex align-items-center justify-content-between"
                  href="#"
                >
                  Restart WordPress
                  <i
                    aria-hidden="true"
                  >
                    <svg
                      class="d-block"
                      fill="currentColor"
                      height="1em"
                      stroke="currentColor"
                      stroke-width="0"
                      viewBox="0 0 448 512"
                      width="1em"
                      xmlns="http://www.w3.org/2000/svg"
                    >
                      <path
                        d="M400 32H48C21.5 32 0 53.5 0 80v352c0 26.5 21.5 48 48 48h352c26.5 0 48-21.5 48-48V80c0-26.5-21.5-48-48-48z"
                      />
                    </svg>
                  </i>
                </a>
              </div>
            </div>
          </div>
        </div>
      </section>
    </main>
    <footer
      class="text-center"
    >
      <p
        class="m-0 common-footer-copyright"
      >
        <small
          class="d-flex align-items-center justify-content-center"
        >
          ©2021 Shifter | A Product by 
          <a
            href="https://www.digitalcube.jp/"
            rel="nofollow noopener"
            title="DigitalCube"
          >
            <img
              alt=""
              class="d-block"
              height="15"
              width="77"
            />
          </a>
        </small>
      </p>
    </footer>
  </div>
</DocumentFragment>
`;
